### ISSUE TITLE
[UIbug] There is no correct display code area

### ISSUE THREAD

**Issue by Dcr-chen** [BloopAI/bloop#631]
When I switch from the default window to the maximization window, the code display area does not follow the maximization.

![image 1](https://github.com/BloopAI/bloop/assets/121916328/e5c1601b-8b8b-446e-80d0-770d02e0f4cf)
![image 2](https://github.com/BloopAI/bloop/assets/121916328/c85bb736-5184-4934-b406-2e3358c2cde0)

**Comment by Dcr-chen** [comment]
Append :  After this bug occurs, it will cause the APP to crash

**Comment by anastasiya1155** [comment]
@Dcr-chen you are right, there is no window resize listener to adjust the code area size. We will change the code display in the next release so it should be fixed. As for the crash, it is very unlikely that resizing the app can cause the crash. Can you provide more details about this?

**Comment by Dcr-chen** [comment]
Crashes don't happen every time, they're accidental, and there's another bug that might have something to do with the listener, and there's a video that shows this problem, when I maximize and then restore the default window size, I can't switch files in Explorer

https://github.com/BloopAI/bloop/assets/121916328/08bcf558-b51c-4d60-ae7d-27bbe7c8a0c2

**Comment by anastasiya1155** [comment]
@Dcr-chen are you sure it only happens after maximizing the window?

**Comment by Dcr-chen** [comment]
Yes, since I updated to 0.4. 4 yesterday, there has been a crash

**Comment by anastasiya1155** [comment]
did you have the ability to submit the crash report? There should be a popup on the Frontend asking you to write a comment and submit a crash report. Or was it completely broken so the app exited and had to be restarted?
<img width="500" alt="Screenshot 2023-06-13 at 09 42 43" src="https://github.com/BloopAI/bloop/assets/35258279/1865e14c-53c1-4bf4-9506-af4994793867">

**Comment by Dcr-chen** [comment]
It just crashed and disappeared, and then a system window popped up asking me: Is bloop working properly? My system is win11, but Bloop didn't catch an error message
